suite/tests/client-interface/reg_size_test.dll.c

```diff
@@ -0,0 +1,38 @@
+#include "dr_api.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+DR_EXPORT void
+dr_client_main(client_id_t id, int argc, const char *argv[])
+{
+    dr_set_client_name("DynamoRIO Sample Client 'test_reg_size'",
+                       "http://dynamorio.org/issues");
+
+    opnd_size_t res;
+    for (int i = DR_REG_NULL + 1; i < DR_REG_LAST_VALID_ENUM; ++i) {
```

Comment:
Fencepost error since the last is inclusive: `s/</<=/`

Reply:
(Looks like this was done so I will resolve it.  The convention is for you to reply Done.)